hack/make-rules/verify/shellcheck.sh

```diff
@@ -34,7 +34,7 @@ SHELLCHECK_IMAGE='koalaman/shellcheck:v0.7.1'
 all_shell_scripts=()
 while IFS=$'\n' read -r script;
   do git check-ignore -q "$script" || all_shell_scripts+=("$script");
-done < <(grep -irl '#!.*sh' . | grep -Ev '^(\./\.git/)|(\./vendor/)|(\./bin/)')
+done < <(grep -irl '#!.*sh' . | grep -Ev '(^\./\.git/)|(^\./vendor/)|(^\./bin/)|(\.go$)')
```

Comment:
this is equivalent to before, but ignoring files ending in `.go` (we don't just match `.sh` in the case of e.g. extensionless scripts like `clean-install`)

Reply:
(ignoring because it tries to parse the go files as bash, not because it has errors in the bash in the go fiels)